### ISSUE TITLE
don`t work with flux-fill svdq

### ISSUE THREAD

**Issue by Daytone29** [lrzjason/T2ITrainer#21]
please help.My error:
 Error loading pipeline: Target modules {'add_v_proj', 'add_k_proj', 'to_q', 'to_add_out', 'add_q_proj', 'to_k', 'to_v', 'to_out.0'} not found in the base model. Please check the target modules and try again.

#config.json 
{
    "script": "train_flux_lora_ui_with_mask.py",
    "seed": 4321,
    "mixed_precision": "bf16",
    "report_to": "wandb",
    "lr_warmup_steps": 0,
    "output_dir": "/home/taras/T2ITrainer/lora",
    "save_name": "flux-lora",
    "train_data_dir": "/home/taras/T2ITrainer/images",
    "optimizer": "adamw",
    "lr_scheduler": "constant",
    "learning_rate": 0.0001,
    "train_batch_size": 1,
    "repeats": 10,
    "gradient_accumulation_steps": 1,
    "num_train_epochs": 5,
    "save_model_epochs": 1,
    "validation_epochs": 1,
    "rank": 4,
    "skip_epoch": 0,
    "skip_step": 0,
    "gradient_checkpointing": true,
    "validation_ratio": 0.1,
    "pretrained_model_name_or_path": "/home/taras/T2ITrainer/flux_models/fill",
    "model_path": "",
    "resume_from_checkpoint": "",
    "recreate_cache": false,
    "config_path": "/home/taras/T2ITrainer/config.json",
    "resolution": "1024",
    "caption_dropout": 0.1,
    "cosine_restarts": 1,
    "max_time_steps": 0,
    "blocks_to_swap": 15,
    "mask_dropout": 0,
    "reg_ratio": 0.7
}

#pipeline
transformer = NunchakuFluxTransformer2dModel.from_pretrained("mit-han-lab/svdq-int4-flux.1-fill-dev", offload=True)
    text_encoder_2 = NunchakuT5EncoderModel.from_pretrained("mit-han-lab/svdq-flux.1-t5")

    pipe = FluxFillPipeline.from_pretrained(
        "black-forest-labs/FLUX.1-Fill-dev",
        transformer=transformer,
        text_encoder_2=text_encoder_2,
        torch_dtype=torch_dtype,
    ).to(device)
    pipe.load_lora_weights('/T2ITrainer/lora/flux-lora-900', weight_name='pytorch_lora_weights.safetensors')
    pipe.fuse_lora()
